core/block.js

```diff
@@ -271,6 +271,23 @@ Blockly.Block.prototype.dispose = function(healStack) {
   }
 };
 
+/**
+ * Call initModel on all fields on the block.
+ * May be called more than once.
+ * Either initModel or initSvg must be called after creating a block and before
+ * first interaction with it.
+ * @public
+ */
+Blockly.Block.prototype.initModel = function() {
```

Comment:
Why not call input.init() and remove the rendered check in Input.init since the comment in that file is wrong?

Reply:
nevermind. init and initModel are very different things.